h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -471,8 +471,10 @@ public Vec vec() {
   public String[][] domains() {
     Vec[] vecs = vecs();
     String ds[][] = new String[vecs.length][];
-    for( int i=0; i<vecs.length; i++ )
-      ds[i] = vecs[i].domain();
```

Comment:
I don't understand this fix change - domain() returns null if the Vec is not categorical. This should not modify the behavior

Reply:
That is where the crash has been, null pointer exception for all non-enum columns.  By checking and making sure the column is enum, I avoided NPE error.

Reply:
I think the NPE is caused by the Vec being null.